packages/bee-q/src/components/dialog/bq-dialog.types.ts

```diff
@@ -0,0 +1,5 @@
+export const DIALOG_SIZE = ['small', 'medium', 'large'] as const;
+export type TDialogSize = (typeof DIALOG_SIZE)[number];
+
+export const DIALOG_FOOTER_VARIANT = ['standard', 'light'] as const;
```

Comment:
I would recommend using `highlight` as a value for the footer appearance when it has a background color gray.

```suggestion
export const DIALOG_FOOTER_VARIANT = ['standard', 'highlight'] as const;
```

Reply:
Done